oemof/solph/components.py

```diff
@@ -820,25 +796,13 @@ def _Q_min_res_rule(block, n, t):
     def _objective_expression(self):
         r"""Objective expression for generic CHPs with no investment.
 
-        Note: This adds only fixed costs as variable costs are already
+        Note: This adds nothing as variable costs are already
         added in the Block :class:`Flow`.
         """
         if not hasattr(self, 'GENERICCHPS'):
             return 0
 
-        fixed_costs = 0
-
-        m = self.parent_block()
-
-        for n in self.GENERICCHPS:
-            if n.fixed_costs is not None:
-                P_max = [list(n.electrical_output.values())[0].P_max_woDH[t]
-                         for t in m.TIMESTEPS]
-                fixed_costs += max(P_max) * n.fixed_costs
-
-        self.fixed_costs = Expression(expr=fixed_costs)
-
-        return fixed_costs
+        return 0
```

Comment:
If you agree, we could also remove the objective expression completely as it holds nothing!?

Reply:
It's your decision @uvchik. For the rest, I am fine. Thanks!

Reply:
@ckaldemeyer  you can remove the 'empty' method, just make sure that no error occurs, as for all blocks the method is automatically called

Reply:
> @ckaldemeyer you can remove the 'empty' method, just make sure that no error occurs, as for all blocks the method is automatically called

Let's wait what @uvchik says. But I can do it!